### ISSUE TITLE
Incognito mode icon is glitched in the tab center

### ISSUE THREAD

**Issue by meandavejustice** [bwinton/VerticalTabs#230]
_filing on behalf of @Qhue from https://github.com/mozilla/testpilot/issues/857_

> By going into incognito mode on Firefox, the incognito mode icon is glitched in the tab center.

**Comment by wresuolc** [comment]
Glitched how?  It looks fine to me in nightly.

![private](https://cloud.githubusercontent.com/assets/16946141/15297815/f2990b2e-1b50-11e6-8c13-46d48f4e5c54.png)

**Comment by meandavejustice** [comment]
@Qhue Could you provide a screenshot?

**Comment by 6a68** [comment]
Here's how a new private window looks on OS X, note the mask icon isn't positioned to the left of the 'new tab' UI at top left:

<img width="1332" alt="screen shot 2016-05-16 at 10 40 37 am" src="https://cloud.githubusercontent.com/assets/96396/15298147/c73caa24-1b52-11e6-889a-5bc7df3f5db7.png">

Here's how a private window looks not on OS X, not the new tab page:

<img width="1332" alt="screen shot 2016-05-16 at 10 42 43 am" src="https://cloud.githubusercontent.com/assets/96396/15298186/f77bec9a-1b52-11e6-96ff-b066bd392236.png">

**Comment by 6a68** [comment]
Maybe the mask positioning is a quirk of linux vs os x layout? I don't see any issues either.

**Comment by Qhue** [comment]
[image: private_browsing.PNG]This it what I see when I enter incognito
mode- you can see that the incognito icon that usually appears in the top
right corner of the browser is instead in the center of the tab center area.

**Comment by johngruen** [comment]
@Qhue can you add that image again? It didn't upload. You should just be able to drag and drop into the github comment box. Thanks

**Comment by Qhue** [comment]
![private_browsing](https://cloud.githubusercontent.com/assets/12704804/15325583/c0860ccc-1bff-11e6-9345-e398cea23a4a.PNG)

**Comment by Qhue** [comment]
You can see that there is a slight problem with the mask icon in the tab center. Instead of going into the top right hand corner of the browser, it is in the tab center area.

**Comment by johngruen** [comment]
@Qhue thanks! We're on it.

**Comment by bwinton** [comment]
@Qhue: Can I ask if you're using a Firefox theme, and if so, which one?  (Also, which operating system are you using?  Linux?  Windows 7?)  Thanks!

**Comment by Qhue** [comment]
@bwinton Yes I am using Windows 7 - and yes I am using a theme: https://addons.mozilla.org/en-US/firefox/addon/conan-edogawa-with-snowboar/?src=ss

**Comment by Zaggy1024** [comment]
I see this on Windows 10 32-bit Nightly:
![firefox_2016-06-02_18-05-44](https://cloud.githubusercontent.com/assets/3149592/15763926/af52e048-28ec-11e6-855e-703fcefec644.png)

It doesn't appear that the original report had the OS specified, so perhaps this is what they were talking about?

**Comment by ericawright** [comment]
![screen shot 2016-07-18 at 5 12 29 pm](https://cloud.githubusercontent.com/assets/10803178/16956451/79717b0e-4da6-11e6-86c8-5df1672862d9.png)
![screen shot 2016-07-18 at 5 12 19 pm](https://cloud.githubusercontent.com/assets/10803178/16956460/7e3d6896-4da6-11e6-9687-eb89b15166fe.png)

On Ubuntu after find has been added.

**Comment by thunderdome2** [comment]
Hello.

I'm using Firefox under Debian Jessie with KDE (in french language) as desktop environment. As you on this screenshot, the Incognito mode icon is not visible:
![image](https://cloud.githubusercontent.com/assets/27619936/25803082/92329692-33f5-11e7-8fba-1fb3f39330ad.png)
A workaround could be changing the address bar color, like on Chrome. But I suppose this change is not concerning only the Tab Center addon, but Firefox by itself.
